test/src/e2e_vm_tests/test_programs/should_pass/require_contract_deployment/caller_auth_test/Forc.toml

```diff
@@ -6,4 +6,5 @@ name = "caller_auth_test"
 
 [dependencies]
 auth_testing_abi = { path = "../../test_abis/auth_testing_abi" }
-std = { path = "../../../../reduced_std_libs/sway-lib-std-assert" }
+std = { path = "../../../../../../../sway-lib-std" }
```

Comment:
If `sway-lib-std-assert` doesn't contain everything this test needs, then maybe there's a case for creating a new reduced std library containing stuff related to contract calls and deployment?

Probably best to do that as a separate issue.